src/main/java/edu/project1/Dictionary.java

```diff
@@ -0,0 +1,27 @@
+package edu.project1;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+class Dictionary {
+    private static final List<String> DICTIONARY = new ArrayList<>(List.of("machine", "house", "theatre"));
```

Comment:
`BUG` Нуу можно тогда просто
```suggestion
    private static final List<String> DICTIONARY = List.of("machine", "house", "theatre");
```

Reply:
List.of же вернет immutable коллекцию, в которую потом нельзя будет добавить новые слова

Reply:
Да, прости, просто не увидел, чтобы ты где-то в Runtime добавлял новые слова)
Поэтому и попросил сделать DICTIONARY константой и просто List'ом

**Создание со звёздочкой**
Попробуй сделать Dictionary полностью Immutable, при этом не потерям возможность добавление новых слов в Runtime, 
т.е. создавать новый экземплять Dictionary при его обновлении
(будет минишаг в многопоточку)

Reply:
Ну кстати со многопоточкой в джаве я встречался когда делал чат приложение (там нет вебсокетов, чисто подключение через java.net.socket), а в котлине когда делал андроид приложение, но в итоге в практике я использовал лишь супер ограниченную часть Thread классов

Reply:
Нуу на практике мало кто работает с многопточкой) Kotlin = 💕

Reply:
Кстати насчет котлина, почему им не особо много пользуются, если он поприятнее джавы будет, та еще и в нем можно спокойно использовать Java классы?

Reply:
Им пользуются))
Просто не все хотят учить что-то новое + не всегда оправданы переезды на новый язык(особенно если проект не сильно развивается)
Но в Таньке 30% всех Java/Kotlin проектов на котлине

Reply:
Если сильно хочешь, можешь у @sanya_rnd спросить можно ли тебе использовать Kotlin, если проверяющий дз Ассистент не против